### DISCUSSION TITLE
搞不明白的错误，请指导

### DISCUSSION THREAD

**Discussion by q2051839** [1715173329/UnblockNeteaseMusic#949] [Q&A]
INFO: (app) HTTP Server running @ http://0.0.0.0:8080
node:events:491
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8081
    at Server.setupListenHandle [as _listen2] (node:net:1485:16)
    at listenInCluster (node:net:1533:12)
    at Server.listen (node:net:1621:7)
    at C:\server-enhanced\precompiled\app.js:1687:99
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1512:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  code: 'EADDRINUSE',
  errno: -4091,
  syscall: 'listen',
  address: '::',
  port: 8081
}

**Comment by 1715173329** [comment]
8081 端口被其他程序占用了。

**Reply by q2051839** [reply]
我想起来了。我的DNS监听了8081 请问在哪个参数里修改监听。我没找到
	name: 'unblock-netease-cloud-music',
	description: '点亮网易云音乐灰色歌曲',
	script: './app.js', // 入口文件路径
	//scriptOptions:'-p 52100:52101', // 可选参数示例: 自定义端口并开启HTTPS
	// scriptOptions: '-o qq', // 可选参数
	wait: '1', // 程序崩溃后重启时间间隔
	grow: '0.25', // 重启等待时间成长值，第一次1秒，第二次1.25秒。。。
	maxRestarts: '40', // 60秒内最大重启次数
	env: [
这个好像不是
